### PR TITLE
Spanish (Uruguay) strings

### PR DESCRIPTION
**Pull request by Iruxos**

### CODE REVIEW

File: RCa01440
Comment (redddcyclone) [inline RCa01440:1]: Can you delete this binary file? It's not necessary

**Review by redddcyclone — CHANGES_REQUESTED** [review]

**Comment by redddcyclone** [comment]
Hi @Iruxos 

First of all, thanks for the contribution!

As for the commit, I see there is an unnecessary binary file called `RCa01440`. Could you remove it?

I will merge it as soon as possible after the removal.

Thanks!

File: RCa01440
Comment (Iruxos) [reply RCa01440:1]: Deleted!

**Review by redddcyclone — APPROVED** [review]

File: GLPI-AgentMonitor.rc [hunk 1]
```diff
@@ -613,6 +613,100 @@ END
...
+IDS_RMENU_EXIT          "Salir"
+IDS_FILE                "Archivo"
+IDS_DIR                 "Directorio"
+IDS_ERR_MAINWINDOW      "Error al crear la ventana principal de GLPI Agent Monitor"
+IDS_RMENU_NEWTICKET     "Crear Ticket..."
+IDS_ERR_SERVER          "Error al obtener la URL del servidor de GLPI\nDeberá reinstalar el agente."
+IDS_NOTIF_NEWTICKET_TITLE "Se tomó captura de pantalla"
+IDS_NOTIF_NEWTICKET     "Presionar CTRL+V en la ventana de nuevo ticket, o adjuntar la captura como archivo en el formulario."
+IDS_NEWTICKET           "Crear ticket..."
+IDS_VIEWLOGS            "Ver registros del agente"
```
Comment (redddcyclone) [inline GLPI-AgentMonitor.rc]: This string is too big for the "View logs" button. Can you trim it down?

image

**Review by redddcyclone — CHANGES_REQUESTED** [review]

**Comment by redddcyclone** [comment]
@Iruxos 

I added another change request to the code, could you resolve this?

The string is too big for the button.

File: GLPI-AgentMonitor.rc (hunk 1 above)
Comment (Iruxos) [reply GLPI-AgentMonitor.rc]: Done!

**Review by redddcyclone — APPROVED** [review]

**Comment by redddcyclone** [comment]
Thanks for the contribution @Iruxos!

**Pull request merged**